syft/internal/fileresolver/chroot_context.go

```diff
@@ -140,6 +140,40 @@ func (r ChrootContext) ToNativePath(chrootPath string) (string, error) {
 	return responsePath, nil
 }
 
+func (r ChrootContext) ToNativeGlob(chrootPath string) (string, error) {
```

Comment:
Should this just be part of the directory resolver, in the internal `requestGlob` function rather than a semi-exported function on this type? I can see arguments for either spot, so nothing to change.

Reply:
this is in internal though, so technically nothing is being exported. Also it's consistent relative to where we reason about `ToNativePath` today as well.